ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -120,6 +119,7 @@ Select Model Server
     [Documentation]    If there are multiple model servers to choose from, this keyword
     ...    Selects the given one
     [Arguments]    ${model_server}
+    Sleep    1s     reason=It's taking a little bit time to load the Model Server options
```

Comment:
Isn't there some possibility to wait until some particular element is present/enabled on the page instead of this 1 sec wait?

Reply:
+1, sleep risks to be unstable

Reply:
@jstourac @bdattoma modified in order to use an active wait. 
Tested in possitive and negative cases.
Thanks for pointing it ;)